samples/monster_generated.h

```diff
@@ -6,6 +6,9 @@
 
 #include "flatbuffers/flatbuffers.h"
 
+#include <string>
```

Comment:
why were these added? `flatbuffers.h` already pulls these in.

Reply:
they were just in the tests/generate_code.{sh,bat} - i removed the samples